### ISSUE TITLE
Create "Join Us" page on the website

### ISSUE THREAD

**Issue by emadvote** [tcfev/forDem#82]
# User Story

## Available Positions
As a guest I want to see a list of available positions in Fordem project on top of this page. Each item list of available positions should include these information:
- Job Title
- Location
- Language
- Terms
- Salary / Rate
- About the role
- Responsibilities
- Candidate Requirements

## Application Form
As a guest I should be able to fill a form of contributor in the website. The form can be a Google Form or a developed form from scratch. The required fields for this form are:
- Name 
-- Type: Text Field
-- Placeholder: Name
-- Label: Name
-- HelperText: can be a nickname or fake name
- Email 
-- Type: Email
-- Placeholder: Email Address
-- Label: Email Address
-- HelperText: Do not use your main email address
- Preferred Position
-- Type: Select
-- Placeholder: Select your preferred position
-- Label: Preferred Position
- Commitment
-- Type: Text Field
-- Validation: only number
-- Name: Commitment
-- Helper Text: How many hours per week can you contribute?
-- Placeholder: Type the number of hours
- Language
-- Type: Checkbox
-- Placeholder: Select your languages and level of proficiency
-- options: Persian, English, German, Other
-- level: Native / Bilingual Proficiency, Full Professional Proficiency, Professional Working Proficiency, Limited Working Proficiency
- Acknowledgment
-- Type: Checkbox
-- Label: I have read the [fordem](https://fordem.org/terms) terms and conditions and I accept all it's terms.

**Comment by armantorkzaban** [comment]
Do we need to work on this for now? @zanzendegi.
If not, let us close it or move it out of the sprint and close.

**Comment by armantorkzaban** [comment]
@MarleneFuerDem @desputinski this also needs to be added to the website as a menu entry. Could you take of care of it?

**Comment by desputinski** [comment]
Not sure. The website seems to be down. And I don't have a login: https://fordem.org/

**Comment by armantorkzaban** [comment]
No, the website is not down. We just plugged the app to the address of the website.
I do not know what the plan of @zanzendegi is, but I assume that we will have a view for non-users where we could have a link at the bottom pointing at 'About/blog/etc.'

**Comment by zanzendegi** [comment]
Hi,
Currently fordem.org is used for the web app. 

The landing page is located at about.fordem.org.
